aws/fargate/README.md

```diff
@@ -0,0 +1,40 @@
+Building And Pushing Docker Image For Initial AWS Deployment of Dev Infrastructure
+---------------------------
+When deploying the initial AWS infrastructure for the Dev environment,
+you need to build and push the Docker image to AWS ECR manually before setting up the ECS services and CI/CD pipeline.
+
+Building Docker Image
+---------------------------
```

Comment:
Vhange headers consistently to the dame format.
(I prefer all #:s, but choose whatever one header style)